### ISSUE TITLE
ReactPdf.renderToString generates an empty white pdf

### ISSUE THREAD

**Issue by tomaswitek** [diegomura/react-pdf#1067]
**Describe the bug**

It seems like there is something wrong with encoding while calling `renderToString` because `renderToFile` and `renderToStream` seems to works ok with the same code.

**File comparison:**
<img width="1458" alt="react-pdf-next" src="https://user-images.githubusercontent.com/340649/100332839-2aa49980-2fd2-11eb-92c2-db66fd28c848.png">

The stream part seems to be different.
Maybe there is a just a problem in my `next.js` code.
Although I tried every possible solution I could find.
```
IndexPage.getInitialProps = async ({res}) => {
  const filename = "file.pdf";
  await ReactPdf.renderToFile(<PdfDocument />, filename);
  const string = await ReactPdf.renderToString(<PdfDocument />);
  res.setHeader("Content-disposition", `attachment; filename="${filename}"`);
  res.setHeader("Content-Type", "application/pdf");
  res.end(string);
};
``` 
Here is a repo https://github.com/tomaswitek/react-pdf-next to reproduce it.

**To Reproduce**
Steps to reproduce the behavior including code snippet (if applies):
1. `git clone https://github.com/tomaswitek/react-pdf-next`
2. `yarn install && yarn dev`
3. Open [http://localhost:3000](http://localhost:3000)
4. An empty PDF will be downloaded automatically

**Expected behavior**
A valid PDF should be downloaded

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [MacOS]
 - Browser [chrome]
 - React-pdf version [2]

**Comment by diegomura** [comment]
Can you please try if this still happens in 2.0? Thanks in advance!

**Comment by diegomura** [comment]
I was able to validate this with 2.0 as well.
![Screen Shot 2021-04-09 at 1 17 17 PM](https://user-images.githubusercontent.com/5600341/114210299-fce34000-9935-11eb-9b69-61ef25ebcc0f.png)

It seems there's an encoding issue there but couldn't yet find what 😞

**Comment by diegomura** [comment]
> 'utf8' (alias: 'utf-8'): Multi-byte encoded Unicode characters. Many web pages and other document formats use UTF-8. This is the default character encoding. When decoding a Buffer into a string that does not exclusively contain valid UTF-8 data, the Unicode replacement character U+FFFD � will be used to represent those errors.

I think it's related to that. However, I tried other encodings and none seems to work. 

Source: https://nodejs.org/api/buffer.html#buffer_buf_tostring_encoding_start_end

**Comment by diegomura** [comment]
Please check PR comment!

**Comment by sneko** [comment]
@diegomura I'm also getting blank PDFs when using `renderToString` and a buffer with latest version.

After some research I just gave up and switched to `renderToStream` to end with a buffer. Either use this https://github.com/jferrl/stream-to-buffer or for direct code: https://stackoverflow.com/a/67729663/3608410

**Comment by jeetiss** [comment]
@sneko you should try `renderToBuffer` it should work fine

https://github.com/diegomura/react-pdf/blob/master/packages/renderer/src/node/renderTo.js#L27-L36

**Comment by sneko** [comment]
@jeetiss oh... despite I tried to look at all properties I didn't see it... I was mainly looking at types but this one is missing into https://github.com/diegomura/react-pdf/blob/master/packages/renderer/index.d.ts

Thank you,

**Comment by jeetiss** [comment]
yep, i still need to add this method to the docs and deprecate `renderToString`
